raiden/api/rest.py

```diff
@@ -384,15 +384,29 @@ def _is_raiden_running(self):
 
     def _serve_webui(self, file_name='index.html'):  # pylint: disable=redefined-builtin
         try:
-            assert file_name
+            if not file_name:
```

Comment:
Why the change? Just to raise `NotFound` instead of assertion exception?

Reply:
From what I understand now, `AssertionError` should be used exclusively for errors which aren't expected/should be fatal. This change was just to replace _expected_ `assert` by one of the already handled errors (which is also coherent with the logic here)